src/libstd/thread/mod.rs

```diff
@@ -88,43 +86,43 @@
 //!
 //! ## The `Thread` type
 //!
-//! Threads are represented via the `Thread` type, which you can get in one of
+//! Threads are represented via the [`Thread`] type, which you can get in one of
 //! two ways:
 //!
-//! * By spawning a new thread, e.g. using the `thread::spawn` function, and
-//!   calling `thread()` on the `JoinHandle`.
-//! * By requesting the current thread, using the `thread::current` function.
+//! * By spawning a new thread, e.g. using the [`thread::spawn`][`spawn]
+//!   function, and calling [`thread()`] on the [`JoinHandle`].
```

Comment:
Nit: As per the docs meeting this week, we don't need the `()` in the function/method display names for links, but we can address all these later in one PR.